common.props

```diff
@@ -3,8 +3,8 @@
 		<Title>Generic Repository Pattern For Entity Framework Core</Title>
 		<RepositoryType>Git</RepositoryType>
 		<NeutralLanguage>en-US</NeutralLanguage>
-		<TargetFrameworks>net5.0</TargetFrameworks>
-		<Version>1.4.0</Version>
+		<TargetFrameworks>net6.0</TargetFrameworks>
+		<Version>1.4.1</Version>
```

Comment:
These changes do not affect the minor version. Framework changes should be assigned to higher versions.

So use 1.5.0 instead of 1.4.1. 
fyi @jeffward01